MusicControlActions/src/org/jjazz/ui/musiccontrolactions/ControlToolbarPanel.java

```diff
@@ -228,7 +228,14 @@ private void initComponents()
         fbtn_Loop = new org.jjazz.ui.flatcomponents.FlatToggleButton();
         fbtn_PlaybackPoint = new org.jjazz.ui.flatcomponents.FlatToggleButton();
         filler5 = new javax.swing.Box.Filler(new java.awt.Dimension(15, 0), new java.awt.Dimension(15, 0), new java.awt.Dimension(15, 32767));
-        panel_Tempo = new javax.swing.JPanel();
+        panel_Tempo = new javax.swing.JPanel()
+        {
+            @Override
+            public java.awt.Dimension getMaximumSize()
+            {
+                return getMinimumSize();
```

Comment:
Not sure to understand why getMinimumSize() rather than getPreferredSize() ?

Reply:
Assumption was that minimum size adjusts to look&feel unlike to preferred size, but I could be wrong... I don't use BoxLayout often...

Sorry, I can't test it on Mac as I don't have one.